src/daemon/jsonrpc/api.rs

```diff
@@ -1123,6 +1123,21 @@ impl RpcApi for RpcImpl {
 
             let spent_vaults = db_vaults_from_spend(&db_path, &db_spend.psbt.txid())
                 .map_err(|e| internal_error!(e))?;
+
+            // FIXME: a temporary hack for the GUI to be able to use its signer properly. Remove
+            // after the demo release in favour of having the derivation indexes in the PSBT.
+            let mut deposit_outpoints = dep_outpoints.clone();
+            for o in dep_outpoints {
+                let i = db_spend
+                    .psbt
+                    .tx()
+                    .input
+                    .iter()
+                    .position(|i| i.previous_output == o)
```

Comment:
I think it is wrong again, previous output is unvault outpoint not deposit outpoint

Reply:
Yeah as said on SIgnal, brainfart. Should be good now